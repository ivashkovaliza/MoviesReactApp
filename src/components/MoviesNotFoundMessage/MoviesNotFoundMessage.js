import React from "react";
import './MoviesNotFoundMessage.scss';

const MoviesNotFoundMessage = () => {
  return (
    <p className={'not-found-msg'}>No films Found</p>
  );
};

export default MoviesNotFoundMessage;
