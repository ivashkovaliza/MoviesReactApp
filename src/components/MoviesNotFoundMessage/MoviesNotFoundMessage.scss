.not-found-msg {
  padding-top: 30px;
  font-size: 48px;
}
