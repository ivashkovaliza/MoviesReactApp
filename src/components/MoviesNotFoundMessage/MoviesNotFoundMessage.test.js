import React from 'react';
import MoviesNotFoundMessage from './MoviesNotFoundMessage';

describe('MoviesNotFoundMessage', () => {
  test('renders correctly', () => {
    const component = shallow(
      <MoviesNotFoundMessage />
    );
    expect(component).toMatchSnapshot();
  });
});
