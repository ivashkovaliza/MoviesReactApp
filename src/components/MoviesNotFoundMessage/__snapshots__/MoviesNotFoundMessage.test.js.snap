// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MoviesNotFoundMessage renders correctly 1`] = `
<p
  className="not-found-msg"
>
  No films Found
</p>
`;
